collect_app/src/main/java/org/odk/collect/android/preferences/ServerPreferencesFragment.java

```diff
@@ -211,8 +211,8 @@ public void onPause() {
     public boolean onTouch(View v, MotionEvent event) {
         final int DRAWABLE_RIGHT = 2;
         if (event.getAction() == MotionEvent.ACTION_UP) {
-            if (event.getX() >= (v.getWidth() - ((EditText) v)
-                    .getCompoundDrawables()[DRAWABLE_RIGHT].getBounds().width())) {
+            int width = ((EditText) v).getCompoundDrawables()[DRAWABLE_RIGHT].getBounds().width();
```

Comment:
What exactly is this the width of? I like introducing a var here, it'd be even better if it explained what this is.